mkposters/mkposter.py

```diff
@@ -42,8 +70,15 @@ def mkposter(datadir):
         html_out = rf"""<!doctype html>
         <html>
         <head>
+        <body style="background-color:{background_color}">
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback">
         <link rel="stylesheet" type="text/css" href="style.css"/>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/styles/{code_style}.min.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/highlight.min.js"></script>
+        <script>hljs.highlightAll();</script>
```

Comment:
Can these two lines be optional (based on a particular magic value of `code_style`) and bump the minor version number (as this is backward-incompatible).

Reply:
Sure good point. On some browsers the superfences just wasn't working so added the `highlight.js` which seemed to do the trick.

Reply:
Right, different browsers have pretty different behaviour. There's a reason I mandated specifically Firefox in the README, as nothing else seemed to work reliably.

Reply:
Great, that was my impression as well. I honestly would prefer to avoid starting down the yaml path and would rather keep it as simple as possible from a user experience perspective. I like the idea of an optional user specified scss, which I can bundle in this PR.

Thanks again for sharing this work - I've demoed an internal project using the html rendered with mkposters and find it useful for quickly creating a poster style presentation given my rusty html. I've also shared the `torchsde` -> `diffrax` work in a couple internal Lunch 'n Learns which, along with your thesis, are awesome.

Reply:
Made code style and mermaid optional for backwards compatibility in latest commit.